tests/settings_tests/tests.py

```diff
@@ -148,6 +148,7 @@ class SettingsTests(SimpleTestCase):
     def setUp(self):
         self.testvalue = None
         signals.setting_changed.connect(self.signal_callback)
+        self.msg = "'Settings' object has no attribute %r"
```

Comment:
That can be a class-level attribute -- no need to assign it in `setUp` for each test. Probably should have a more descriptive name like `no_attribute_msg`.